src/Command/InfectionCommand.php

```diff
@@ -324,6 +328,61 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
         $this->eventDispatcher = $this->container['dispatcher'];
     }
 
+    private function initContainer(InputInterface $input): void
+    {
+        /** @var string|null $configFile */
+        $configFile = $input->hasOption('configuration')
+            ? trim((string) $input->getOption('configuration'))
+            : null
+        ;
+
+        if ($configFile === '') {
+            $configFile = null;
+        }
+        /** @var string $existingCoveragePath */
+        $existingCoveragePath = $input->hasOption('coverage')
+            ? trim((string) $input->getOption('coverage'))
+            : ''
+        ;
+        /** @var string $initialTestsPhpOptions */
+        $initialTestsPhpOptions = trim((string) $input->getOption('initial-tests-php-options') ?: '');
+        /** @var bool $ignoreMsiWithNoMutations */
+        $ignoreMsiWithNoMutations = $input->getOption('ignore-msi-with-no-mutations');
+        $minMsi = (float) $input->getOption('min-msi');
+        $minCoveredMsi = (float) $input->getOption('min-covered-msi');
+        $mutators = $input->hasOption('mutators')
```

Comment:
As of now normalizing this input is kinda verbose. I think it would make sense to have a normalizer of some sorts around the `InputArgument` and `InputOption`, but I prefer to check that out later rather than here